### ISSUE TITLE
Irrigation sensor = 0

### ISSUE THREAD

**Issue by liouma** [jeroenterheerdt/HAsmartirrigation#445]
### What happened?

I have a drip watering system connected on a zigbee valve and I just need to calcultate once per day the time to open the valve.
I have a weather station in the garden collecting many parameters but not evapotranspiration, so I expect it to be calculated by smart irrigation.
The weather forecast is pretty reliable on 48h.

I configured the integration as described bellow and irrigation sensor stay to 0 min.

### How to reproduce

My config:

General:
Weather data automatic : yes
Update sensorrs every : 1 hour
delay : 0

Automatic time calculation per zone : Yes
at: 17:55

Automatic suppression of weather data: Yes
at 23:59

Continuous updates for sensors (experimental) : No

ZONES (only one)
![image|507x285](upload://fbi3tu79KQMjyWYRDzxk4BL3xs.png)
As it's a drip irrigation system on the feet of the plants I don't have a big area but I estimeted it to 15m² and I added a multiplier x10
I'm not sure about the bucket settings. Is it automatic? should I fill it?

MODULES
![image|541x186](upload://9Q3ic5EruRtNXxYMUrfip57SkuT.png)

SENSOR GROUPS (only one):
Dew point: something strange happened here many times. When I load the page instead of the full name of the sensor, there is just "senso". I write the full name of the sensor, it seem ok. When I come back the next day there is only "senso".
Evapotranspiration: NONE because I don't have a sensor, I guess it will be calcultated.
Here is my full settings:
![image|245x500](upload://mWirl25ORydxFQuV4whTuP0OE9p.png)

[config_entry-smart_irrigation-6009db890df4992f4f9372f720658638.json](https://github.com/user-attachments/files/16285459/config_entry-smart_irrigation-6009db890df4992f4f9372f720658638.json)
[home-assistant_smart_irrigation_2024-07-18T15-33-36.071Z.log](https://github.com/user-attachments/files/16285582/home-assistant_smart_irrigation_2024-07-18T15-33-36.071Z.log)

### Relevant log output

```shell
I went on the interface registering the logs. Modified many times the field dew point with value : sensor.station_meteo_dew_point
Everytime I refresh the value is "senso".
I stopped registering the logs and uploaded the file.
```

### Which version are you running?

v2024.7.5

### Diagnostics file

- [X] I have attached a diagnostics file

### Additional information

I'll be happy to provide any useful additional information

**Comment by liouma** [comment]
I deleted the sensor group and managed to recreate it. All values seem OK.
I hope this will work better

**Comment by jeroenterheerdt** [comment]
it's weird. your pressure and dewpoint sensors are / were incorrect. what are the entity names for these? I wonder if there is a encoding issue since you are using a french install (?).

**Comment by liouma** [comment]
Pressure sensor:
name "Pression"
sensor.station_meteo_pression

Dew point:
name: "Station météo Dew point"
sensor.station_meteo_dew_point

There is the letter 'é' in the second, but I have hundreds of sensors using this letter without problem.

**Comment by jeroenterheerdt** [comment]
Yeah, that weird. Is there any chance you can give me the full list of sensors you want to use so I can mimick your install and try to find a bug?

**Comment by liouma** [comment]
I kept playing with the settings. Now I have "sensor group2" that seem fully operational and it's confirmed as it just rendered a sensor = 3 427s
The bucket has been set to -1,5. Is it normal ?
![image](https://github.com/user-attachments/assets/165d5222-9962-4616-af0a-8a32c9a3557f)

As I destroyed the non working sensor groups, I'm sorry but I won't be able to give you more elements about the bug.
I'll check tomorow what's happening and let you know if it work or not.
